AdvancedCalculator/Infrastucture/Calculators/ExpandedCaclulator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AdvancedCalculator.Infrastucture.Calculators
+{
+    public class ExpandedCaclulator : BaseCalculator
+    {
+        public bool FirstNumber(int number)
+        {
+            bool FirstN = true;
+            for (int i = 2; i < number; i++)
+            {
+                if(number % i == 0)
+                {
+                    FirstN =  false;
+                    break;
+                }
+                         
+            }
+            if(FirstN == true)
+            {
```

Comment:
Myślę że w tym przypadku możemy też usunąc te nawiasy. Nie będzie to aż tak bardzo kolidowało w czytelność a będzei mniej kodu